phpstan.types.neon.dist

```diff
@@ -0,0 +1,4 @@
+parameters:
+  paths:
+    - types
```

Comment:
Should we move this file to the `tests/` folder?

Reply:
No.